doc/usage/restructuredtext/directives.rst

```diff
@@ -708,6 +708,21 @@ __ http://pygments.org/docs/lexers
    .. versionchanged:: 2.1
       Added the ``force`` option.
 
+
+.. rst:directive:: .. docinclude:: filename
+
+   Includes another document in this one. Unlike :directive:`include`, this
+   allows inclusion of documents that use different parsers to their parent
+   document, such as including a markdown file in an rst file.
+
+   .. rst:directive:option:: start-line
+                             end-line
+                             start-after
+                             end-after
```

Comment:
Are these filtering options needed? In my experience of `literalinclude` directive, many filter options are requested. So I hesitate to add filtering options.

Reply:
Yes, see https://github.com/pygae/galgebra/pull/413 for a useful application.

These filters are part of the built-in `.. include`, so it seems reasonable to keep them anyway.